router/middleware/logger.go

```diff
@@ -20,6 +20,33 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// This file, in part, reproduces portions of
+// https://github.com/elastic/ecs-logging-go-logrus/blob/v1.0.0/formatter.go
+// to handle our custom fields in Format().
+
+const (
+	// ecsVersion holds the version of ECS with which the formatter is compatible.
+	ecsVersion = "1.6.0"
+)
+
+var (
+	ecsFieldMap = logrus.FieldMap{
+		logrus.FieldKeyTime:  "@timestamp",
+		logrus.FieldKeyMsg:   "message",
+		logrus.FieldKeyLevel: "log.level",
+	}
+)
+
+// Formatter holds ECS parameter information for logging.
+type Formatter struct {
```

Comment:
I would propose a more specific name. Maybe `ECSFormatter`? If/When we add another formatter we will run into issues otherwise.

Reply:
addressed. i actually initially had it this way, but changed it. lesson learned, go with your gut.